web/client/epics/identify.js

```diff
@@ -440,6 +440,36 @@ export const setMapTriggerEpic = (action$, store) =>
             );
         });
 
+/**
+ * Epics for Identify and map info when the time parameter changes from the timeline.
+ * This triggers the featureInfoClick action.
+ */
+export const handleGetFeatureInfoForTimeParamsChange = (action$, {getState}) =>
+    action$
+        // when time is updated...
+        .ofType(CHANGE_LAYER_PARAMS) // CHANGE_LAYER_PARAMS: time params changes, so need update the feature info if the popup is open
+        .filter(({ params = {} }) => {
+            // Only process if params is time and there's a click point
+            const state = getState();
+            return includes(Object.keys(params), "time") && clickPointSelector(state);
+        })
+        .switchMap(({ layer = [], params = {} }) => {
+            const state = getState();
+            const groups = rawGroupsSelector(state);
+            const queryableLayers = reverse(getDerivedLayersVisibility(queryableLayersSelector(state), groups));
+            const filterNameList = queryableLayers.filter(({ id, visibility }) => visibility && layer.includes(id)).map(({ name }) => name);
+            const overrideParams = filterNameList.reduce((obj, filterName) => ({ ...obj, [filterName]: params}), {});
```

Comment:
`OverrideParams` doesn't look to be necessary. I removed it and it works anyway. Did you needed to use it for some reason?
But for other params, I'm not sure they can all be reproduced except this.
I'd suggest to use as sample `onUpdateFeatureInfoClickPoint` epic, that should do more or less what we should do...

Did you tried to do it yet?